numpy/core/src/umath/wrapping_array_method.c

```diff
@@ -240,8 +244,8 @@ PyUFunc_AddWrappingLoop(PyObject *ufunc_obj,
         if (cmp == 0) {
             continue;
         }
-        wrapped_meth = (PyArrayMethodObject *)PyTuple_GET_ITEM(item, 1);
-        if (!PyObject_TypeCheck(wrapped_meth, &PyArrayMethod_Type)) {
+        wrapped_meth = (PyArrayMethodObject *)PyTuple_GetItem(item, 1);
```

Comment:
Not sure why the macro `GET_ITEM` causes problems here, something about the type of `item`?

Reply:
Me either, but using the macro failed on PyPy.